src/store/listDetails/operations/removeUserListOperation.js

```diff
@@ -0,0 +1,36 @@
+import { createLogic } from 'redux-logic'
+
+import { openNotification } from 'Utils'
+import { sessionIdSelector } from 'Store/auth/selectors'
+import { REMOVE_USER_LIST } from '../types'
+
+export const removeUserListOperation = createLogic({
+  type: REMOVE_USER_LIST,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { listId, history } = action
```

Comment:
can we get history inside operation?

Reply:
@VladWithCookies as I understood, no, we cant

Reply:
As I understood no, we cant

Reply:
ok